gradle-module-plugin/src/functionalTest/kotlin/io/ia/sdk/gradle/modl/BaseTest.kt

```diff
@@ -50,25 +48,36 @@ open class BaseTest {
         return name
     }
 
-    protected fun prepSigningResourcesForModuleName(
-        parentDirectory: Path,
-        moduleName: String,
-        withPropFile: Boolean = true
-    ): SigningResources {
-        val projectDir = nameToDirName(moduleName)
-        return prepareSigningTestResources(parentDirectory.resolve(projectDir), withPropFile)
-    }
-
     // writes a gradle.properties file containing the signing credentials needed for signing a module using test
     // resources
-    protected fun writeSigningCredentials(targetDirectory: Path): Path {
+    protected fun writeSigningCredentials(
+        targetDirectory: Path,
+        keystoreProps: String,
+        writeBoilerplateProps: Boolean = true
+    ): Path {
         val gradleProps: Path = targetDirectory.resolve("gradle.properties")
+        gradleProps.toFile().let { propsFl ->
```

Comment:
This file access is awkward to me, but I'll let it slide.
I think there's something here with `buildString` or creating your own `PrintWriter` or something that would make it a bit more clear.